modules/nixos/kitsune-bootstrap.nix

```diff
@@ -0,0 +1,93 @@
+{
+  self,
+  config,
+  lib,
+  pkgs,
+  ...
+}: let
+  cfg = config.services.kitsune-bootstrap;
+in {
+  options.services.kitsune-bootstrap = {
+    enable = lib.mkEnableOption "kitsune-bootstrap";
+
+    package = lib.mkOption {
+      default = self.inputs.holochain.packages.${pkgs.system}.holochain;
+      type = lib.types.package;
+    };
+
+    address = lib.mkOption {
+      description = "address to bind";
+      type = lib.types.str;
+    };
+
+    tls-port = lib.mkOption {
+      description = "port to bind for incoming TLS connections";
+      type = lib.types.int;
+    };
+
+    url = lib.mkOption {
+      description = "url for incoming TLS connections to the bootstrap server";
+      type = lib.types.str;
+    };
+
+    port = lib.mkOption {
+      description = "port to bind";
+      type = lib.types.int;
+    };
+  };
+
+  config = lib.mkIf (cfg.enable) {
+    systemd.services.kitsune-bootstrap = {
+      after = ["network.target"];
+      wantedBy = ["multi-user.target"];
+
+      environment = {
+        TMPDIR = "%T";
```

Comment:
What does this do?

Reply:
it ensures that TMPDIR environment variable is set to something that systemd considers temporary and writable. for some reason it wasn't set otherwise. [systemd.unit](https://www.freedesktop.org/software/systemd/man/latest/systemd.unit.html) or some other manual might explain why.

Reply:
Specifically the `%T` bit looked magical to me